spec/acceptance/fail2ban_spec.rb

```diff
@@ -0,0 +1,273 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+case fact('osfamily')
```

Comment:
that's a legacy fact, can you please try
```suggestion
case fact('os.family')
```

I'm not sure if the dot notation is supported here.